src/utils.js

```diff
@@ -18,3 +20,118 @@ export function splitLocation(location) {
     Building: location,
   };
 }
+
+export function changeStatus(section) {
+  if (section.status === "Open") {
+    return "Add to Calendar";
+  } else if (section.status === "Closed") {
+    return "Section Closed";
+  } else {
+    return `Waitlist (${section.waitlistTotal})`;
+  }
+}
+
+export function styleStatus(status) {
+  if (status === "Open") {
+    return green[500];
+  } else if (status === "Closed") {
+    return red[500];
+  } else {
+    return yellow[800];
+  }
+}
+
+export function parseDate(date) {
+  const datePattern = /^(\d{4})-(\d{2})-(\d{2})\s(\d{1,2}):(\d{2}):(\d{2})$/;
+  const [, year, month, day, rawHour, min, sec] = datePattern.exec(date);
+  return new Date(
+    `${year}-${month}-${day}T${("0" + rawHour).slice(-2)}:${min}:${sec}`
+  );
+}
+
+export function addMinutes(date, minutes) {
+  return new Date(date.getTime() + minutes * 60000);
+}
+
+export function compareTime(timeA, timeB) {
```

Comment:
should this be called `timeIsEqual` instead of `compareTime` since it isn't really a compareTo-type function that returns an int. I feel like it would be a better descriptor of the purpose of the function